src/main/java/com/desiato/puresynth/exceptions/InvalidTokenException.java

```diff
@@ -0,0 +1,7 @@
+package com.desiato.puresynth.exceptions;
+
+public class InvalidTokenException extends Exception {
```

Comment:
Where it is used? Why not use Spring `AuthenticationException`?

Reply:
When you asked about the CustimeAuthenticationException, I also started thinking about the InvalidTokenException. In reality, beyond a more specific name, it does not make a huge difference from AuthenticationException.
I will remove it